api/python/provisioner/commands/__init__.py

```diff
@@ -671,13 +671,21 @@ def dynamic_validation(self, params: inputs.SWUpdateRepo, targets: str):  # noqa
 
             # there is no the same release repo is already active
             if self._is_repo_enabled(f'sw_update_{release}'):
-                raise SWUpdateRepoSourceError(
-                    str(repo.source),
-                    (
-                        f"SW update repository for the release "
-                        f"'{release}' has been already enabled"
-                    )
+                err_msg = (
+                    "SW update repository for the release "
+                    f"'{release}' has been already enabled"
                 )
+                logger.warning(err_msg)
+
+                # TODO IMPROVE later raise and error
+                if False:
```

Comment:
This is a smart way of TODO :)
But why not just leave the actual code commented?

Reply:
Why do we need this logic if we decided to allow the re-uploading of the ISO version? My suggestion, we need to decide what functionality we support. Should we support uploading of SW update of the same version or not? Because in the future, it is possible to get a situation when we will have two possible bugs which contradict each other: one for "we need to allow to re-upload the existing one SW update" and the second one for "We should reject re-uploading of ISO with the existing release".

Reply:
For sure we need that logic but it would require a week of so fo integration with CSM team since they should adjust their logic accordingly.